src/main/java/com/google/sps/travelbud/Event.java

```diff
@@ -26,6 +29,15 @@
     this.location = location;
     this.pricing = pricing;
   }
+  Event(Entity entity) {
+    this.id = (long) entity.getKey().getId();
+    this.name = (String) entity.getProperty("name");
+    this.cityId = (long) entity.getProperty("cityId");
+    this.description = (String) entity.getProperty("description");
+    this.date = (String) entity.getProperty("date");
+    this.location = (String) entity.getProperty("location");
+    this.pricing = (double) entity.getProperty("pricing");
+  }
```

Comment:
Technically there should be a blank line above and beneath this constructor
```suggestion

  Event(Entity entity) {
    this.id = (long) entity.getKey().getId();
    this.name = (String) entity.getProperty("name");
    this.cityId = (long) entity.getProperty("cityId");
    this.description = (String) entity.getProperty("description");
    this.date = (String) entity.getProperty("date");
    this.location = (String) entity.getProperty("location");
    this.pricing = (double) entity.getProperty("pricing");
  }

```